### PR TITLE
Switch to Pulp community releases

### PR DESCRIPTION
**Pull request by ehelms**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.16
* [ ] 1.15
* [ ] 1.14

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: katello/katello-repos/katello.repo [hunk 1]
```diff
@@ -7,14 +7,12 @@ gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-katello
...
 
-# Pulp RPMs as supported by Katello - This is a coordinated
-# copy of Pulp's packages in order to ensure compatibility
-[katello-pulp]
-name=Pulp Community Releases
-baseurl=https://fedorapeople.org/groups/katello/releases/yum/@REPO_VERSION@/pulp/@DIST@/$basearch/
-gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-katello
+[pulp]
+name=Pulp Community Release
+baseurl=https://repos.fedorapeople.org/repos/pulp/pulp/beta/2.15/7Server/$basearch
```
Comment (ekohl) [inline katello/katello-repos/katello.repo]: You can replace ```7Server``` with ```$releasever``` and I wonder if we should replace 2.15 with ```@PULP_VERSION```

File: katello/katello-repos/katello.repo [hunk 2]
```diff
@@ -35,12 +33,12 @@ gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-katello
...
 
-[katello-pulp-source]
-name=Katello Pulp source
-baseurl=https://fedorapeople.org/groups/katello/releases/source/srpm/@REPO_VERSION@/pulp/@DIST@/
-gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-katello
-enabled=0
-gpgcheck=0
+[pulp]
+name=Pulp Community Release Source
+baseurl=https://repos.fedorapeople.org/repos/pulp/pulp/beta/2.15/7Server/src/
```
Comment (ekohl) [inline katello/katello-repos/katello.repo]: Same here as the above baseurl

File: katello/katello-repos/katello.repo [hunk 3]
```diff
@@ -35,12 +33,12 @@ gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-katello
 enabled=0
 gpgcheck=0
 
-[katello-pulp-source]
-name=Katello Pulp source
-baseurl=https://fedorapeople.org/groups/katello/releases/source/srpm/@REPO_VERSION@/pulp/@DIST@/
-gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-katello
-enabled=0
-gpgcheck=0
+[pulp]
```
Comment (ekohl) [inline katello/katello-repos/katello.repo]: pulp-source?

**Comment by ehelms** [comment]
Updated with the suggestions

@jlsherrill will you check this?

**Comment by ehelms** [comment]
Current pipeline failure is now `[ERROR 2018-01-12 15:05:23 verbose]  Execution of '/usr/bin/yum -d 0 -e 0 -y install pulp-deb-plugins' returned 1: Error: Nothing to do` which I think this change would fix instead of updating our Pulp repository.

File: katello/katello-repos/katello.repo [hunk 4]
```diff
@@ -35,12 +33,12 @@ gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-katello
...
-name=Katello Pulp source
-baseurl=https://fedorapeople.org/groups/katello/releases/source/srpm/@REPO_VERSION@/pulp/@DIST@/
-gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-katello
-enabled=0
-gpgcheck=0
+[pulp-source]
+name=Pulp Community Release Source
+baseurl=https://repos.fedorapeople.org/repos/pulp/pulp/@PULP_RELEASE@/@PULP_VERSION@/$releasever/src/
+gpgkey=https://repos.fedorapeople.org/repos/pulp/pulp/GPG-RPM-KEY-pulp-2
+enabled=1
```
Comment (jlsherrill) [inline katello/katello-repos/katello.repo]: this likely should not be enabled by default

**Comment by jlsherrill** [comment]
one small comment, everything else looks fine

**Comment by ehelms** [comment]
[test]

**Comment by ehelms** [comment]
Updated. I had to re-write some of the python in order for it to pass on EL5. The ternary operator being used is not available until Python 2.5.

**Pull request merged**